EGRET.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
No matter what, I'd remove this file from the pull request. If you want, you could add it to the gitignore.